pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "udata-hydra"
 version = "2.0.0.dev"
 description = "Async crawler and parsing service for data.gouv.fr"
 authors = [{ name = "Opendata Team", email = "opendatateam@data.gouv.fr" }]
-requires-python = ">=3.9,<3.13"
+requires-python = ">=3.11,<3.13"
```

Comment:
Is it only due to tomllib that it isn't compatible with 3.9? Do we want to add a if-case to keep 3.9 support (making migration easier maybe).

Reply:
Yes, just because of that. So that we could remove a dependency and use `tomlib` in the standard library.
As you wish, I can't judge yet if we want to favor retrocompatibility towards 3.9, or lighten the dependencies :) Let me know and I'll commit a revert on this precise thing if necessary

Reply:
Since the actual version is already working with 3.11, we can drop 3.9 without making it too hard to upgrade step by step. :ok_hand:

Reply:
In that case I'll create another cleaning PR without the big move to Python 3.11, and rebase that one later one, while we keep that one on pause for now.